test/server/cards/01_SOR/units/BobaFettDisintegrator.spec.js

```diff
@@ -0,0 +1,115 @@
+describe('Boba Fett Disintegrator', function() {
```

Comment:
nit, consistency: comma between title and subtitle - `Boba Fett, Disintegrator`

Reply:
fixed title and subtitle.